test/Altinn.Profile.Tests/IntegrationTests/UserProfileTests.cs

```diff
@@ -144,6 +144,139 @@ public async Task GetUsersById_SblBridgeFindsProfile_ResponseOk_ReturnsUserProfi
             Assert.Equal("nb", actualUser.ProfileSettingPreference.Language);
         }
 
+        [Fact]
+        public async Task GetUsersByUuid_SblBridgeFindsProfile_ResponseOk_ReturnsUserProfile()
+        {
+            // Arrange
+            const int userId = 20000009;
+            Guid userUuid = new("cc86d2c7-1695-44b0-8e82-e633243fdf31");
+
+            HttpRequestMessage sblRequest = null;
+            DelegatingHandlerStub messageHandler = new(async (request, token) =>
+            {
+                sblRequest = request;
+
+                UserProfile userProfile = await TestDataLoader.Load<UserProfile>(userUuid.ToString());
+                return new HttpResponseMessage() { Content = JsonContent.Create(userProfile) };
+            });
+            _webApplicationFactorySetup.SblBridgeHttpMessageHandler = messageHandler;
+
+            HttpRequestMessage httpRequestMessage = CreateGetRequest(userId, $"/profile/api/v1/users/{userUuid}");
+
+            httpRequestMessage.Headers.Add("PlatformAccessToken", PrincipalUtil.GetAccessToken("ttd", "unittest"));
+
+            HttpClient client = _webApplicationFactorySetup.GetTestServerClient();
+
+            // Act
+            HttpResponseMessage response = await client.SendAsync(httpRequestMessage);
+
+            // Assert
+            Assert.NotNull(sblRequest);
+            Assert.Equal(HttpMethod.Get, sblRequest.Method);
+            Assert.EndsWith($"users?useruuid={userUuid}", sblRequest.RequestUri.ToString());
+
+            string responseContent = await response.Content.ReadAsStringAsync();
+
+            UserProfile actualUser = JsonSerializer.Deserialize<UserProfile>(
+                responseContent, serializerOptionsCamelCase);
+
+            // These asserts check that deserializing with camel casing was successful.
+            Assert.Equal(userId, actualUser.UserId);
+            Assert.Equal(userUuid, actualUser.UserUuid);
+            Assert.Equal("LEO WILHELMSEN", actualUser.Party.Name);
+            Assert.Equal(userUuid, actualUser.Party.PartyUuid);
+            Assert.Equal("LEO", actualUser.Party.Person.FirstName);
+            Assert.Equal("nb", actualUser.ProfileSettingPreference.Language);
+        }
+
+        [Fact]
+        public async Task GetUsersByUuid_SblBridgeFindsProfile_NotAuthorized_ReturnsForbidden()
```

Comment:
As I had to remove Authorization to test this in the swagger this test is set in place just to give an error if there is no authorization performed this is to prevent the error I made and Stephanie pointed out. now there will be one or two failing test based on what Authorization Attribute is commented out. If not authorized then it will continue to the SblBridge where it will fail on a InternalServerError and the Assert will flag it as an error as it expect an Authorization failure.